modules/admin/proc-rhdh-deployment-config.adoc

```diff
@@ -96,6 +98,59 @@ spec:
                   claimName: dynamic-plugins-root
 ----
 
+Starting from v1alpha3 (Operator version 0.4.0), you can mount directories from pre-created PVCs using the `spec.application.extraFiles.pvcs` field. PVCs are mounted as directories to the container's path, based on the following logic:
```

Comment:
```suggestion
Starting from v1alpha3, you can mount directories from pre-created PVCs using the `spec.application.extraFiles.pvcs` field. PVCs are mounted as directories to the container's path, based on the following logic:
```

`0.4` is the upstream operator version, but the downstream version is the RHDH version (`1.4`). Not sure we need to specify it here.

Reply:
Removed the version